uint8 role
uint8 id
uint32 local_time
uint32 system_time
float32 voltage
LinktrackTag[] nodes

